### ISSUE TITLE
Consider using Mackup

### ISSUE THREAD

**Issue by underyx** [Strajk/setup#1]
https://github.com/lra/mackup

**Comment by Strajk** [comment]
I'm aware of that project, but don't wanna use it, as it's too much automatic :)

As I'm installing new system about once a year, "semi-automatic" process is good as it forces me to mentally review each step - do I need this app?, is there better preferences available?, ... :))

**Comment by underyx** [comment]
Right, I know what you mean, I actually keep my setups completely manual for
this reason, I don't even keep a reference for myself to make sure I only
install what I really need :D

> On Feb 29 2016, at 12:28 pm, strajk &lt;notifications@github.com&gt; wrote:  

>

> I'm aware of that project, but don't wanna use it, as it's too much
automatic :)

>

> As I'm installing new system about once a year, "semi-automatic" process is
good as it forces me to mentally review each step - do I need this app?, is
there better preferences available?, ... :))

>

> —  
Reply to this email directly or [view it on GitHub](https://github.com/Strajk/
setup/issues/1#issuecomment-190164385).![](https://github.com/notifications/be
acon/ABAuVIoM9ZaCu9SsE-PPnYKVhfYu_JFxks5potF5gaJpZM4Hjze9.gif)
